platform/accessibility/guidance/defect-severity-rubric.md

```diff
@@ -1,102 +1,137 @@
 # Accessibility Defect Severity Rubric
+
 This document will provide a common language for determining accessibility defect seriousness. It will capture the following things:
 
 1. Defect severity
-2. Potential users impacted by the defect
-3. Timeframe in which the defect should be fixed
+2. Timeline to fix the defect. Please note,  `508-defect-0` and `508-defect-1` are launch blockers.
+3. Potential users impacted by the defect
 
-## 508-defect-0 (Showstopper)
+## Table of Contents
+
+* [Defect 0: Showstopper](#508-defect-0)
+* [Defect 1: Critical](#508-defect-1)
+* [Defect 2: Serious](#508-defect-2)
+* [Defect 3: Minor](#508-defect-3)
+* [Defect 4: Trivial](#508-defect-4)
+
+## 508-defect-0
+
+**❗️ Showstopper. Must be fixed immediately.**
+These issues have the potential to cause life-threatening situations and should be be remediated before work starts or continues on `508-defect-1` issues.
+
+### Types of issues:
 
-### Type of issue:
-* Keyboard traps
-* Content is unreachable without a pointer device like a mouse (custom or native elements)
 * [High flicker rates](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html) (could trigger epileptic seizures)
 * Motion or movement that covers a high percentage of the screen (possible [vestibular issues](https://a11yproject.com/posts/understanding-vestibular-disorders/))
 
-### Potential Users Impacted
-* Keyboard navigation
-* Screen reader or other assistive device
-* Users who have partial vision or are blind
-* Users who are hard of hearing or deaf
-* Those who have [vestibular issues](https://a11yproject.com/posts/understanding-vestibular-disorders/) like motion sickness
-* Those who have photosensitivity
-
-### Time to Fix
-* Issue **must be fixed before launch**
-
-## 508-defect-1 (Critical)
-
-### Type of issue:
-* Critical errors in automated testers like [aXe plugin](https://www.deque.com/axe), [SortSite](https://www.powermapper.com/products/sortsite/), [AATT](https://github.com/paypal/AATT)
-* Inaccessible rich content like video without captioning
-* Color contrast issues
-* Missing form labels and explanations
-* Dynamic content being added or removed from the page without announcing a change to assistive devices
-* UI or form controls unreachable by `TAB` or `SHIFT + TAB`
-* Content without a good, clear focus halo
-
-### Potential Users Impacted
-* Keyboard navigation
-* Screen reader or other assistive device
+### Potential impact:
+
+* Users who have photosensitivity
+* Users who have [vestibular issues](https://a11yproject.com/posts/understanding-vestibular-disorders/) like motion sickness
+
```

Comment:
Consider a note that our VA audience has a higher incidence of traumatic brain injury (TBI) and PTSD, and therefore a higher likelihood of these considerations.